OpenRA.Mods.Common/Traits/BotModules/SupportPowerBotModule.cs

```diff
@@ -10,37 +10,60 @@
 #endregion
 
 using System.Collections.Generic;
-using OpenRA.Mods.Common.Traits;
+using System.Linq;
+using OpenRA.Mods.Common.AI;
 using OpenRA.Traits;
 
-namespace OpenRA.Mods.Common.AI
+namespace OpenRA.Mods.Common.Traits
 {
-	class AISupportPowerManager
+	[Desc("Manages bot support power handling.")]
+	public class SupportPowerBotModuleInfo : ConditionalTraitInfo, Requires<SupportPowerManagerInfo>
+	{
+		// TODO Update OpenRA.Utility/Command.cs#L300 to first handle lists and also read nested ones
```

Comment:
This file doesn't exist, and i'm not sure whether this limitation still applies!